internal/actor/user/booklend/v1/commands.proto

```
@@ -1,31 +1,25 @@
 syntax = "proto3";
 
-package actor.booklend.v1;
+package actor.user.booklend.v1;
 
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
-import "google/rpc/code.proto";
+import "eventsourcing/v1/commands.proto";
```

buf

import "eventsourcing/v1/commands.proto": file does not exist

(COMPILE)